src/docs/ar/faq/questions-about-flexpool/xch-does-flexpool-have-transaction-fees.md

```diff
@@ -3,4 +3,4 @@ title: Does Flexpool charge a fee for every transaction?
 coin: xch
 ---
 
-**No**. We don't charge any additional transaction fees from our side. The only fee you will pay is the network transaction fee, that is static regardless of the transaction value.
+**No**. There are no transaction fees paid by the farmer for XCH transactions.
```

Comment:
Right now yes, there are no fees. But in future, highly likely fees will appear as more people will start using Chia.

Reply:
Do you want me just remove this FAQ completely? 
Or I can change the working to "Right now, there are no .......", but seems subtle.

Reply:
Both options will work.

Reply:
Removed